x/tx/textual/bytes_test.go

```diff
@@ -21,7 +21,7 @@ func TestBytesJsonTestCases(t *testing.T) {
 	err = json.Unmarshal(raw, &testcases)
 	require.NoError(t, err)
 
-	textual := textual.NewSignModeHandler(nil)
+	textual := textual.SignModeHandler{}
```

Comment:
Are these changes necessary?

Reply:
I don't think so, I just did it because I saw this comment from you: https://github.com/cosmos/cosmos-sdk/pull/14787#discussion_r1087667093

Reply:
Ah I now see you say in tests, should I revert these and just leave the ones that are in tests?

Reply:
I would say let's not do any `x/tx` changes in this PR, tag a new `x/tx` now, and update in this PR to the new `x/tx`.